src/doc/src/reference/workspaces.md

```diff
@@ -80,10 +80,14 @@ edition = "2021"     # the edition, will have no effect on a resolver used in th
 authors = ["Alice <a@example.com>", "Bob <b@example.com>"]
 ```
 
-Note that in a virtual manifest the [`resolver = "2"`](resolver.md#resolver-versions)
-should be specified manually. It is usually deduced from the [`package.edition`][package-edition]
-field which is absent in virtual manifests and the edition field of a member
-won't affect the resolver used by the workspace.
+When there isn't a root package:
+
+- [`resolver = "2"`](resolver.md#resolver-versions) must be
+  set explicitly in virtual workspaces as they have no
+  [`package.edition`][package-edition] to infer it from.
+  [resolver version](resolver.md#resolver-versions).
+- Commands run in the workspace's directory will run against all workspace
+  members by default, see [Package selection](#package-selection).
```

Comment:
Feel a bit disconnected with the previous paragraph. What about this:

```suggestion
Some major differences bewteen root packages and virtual workspaces:

- In virtual workspaces, [`resolver = "2"`](resolver.md#resolver-versions) must be
  set explicitly as they have no
  [`package.edition`][package-edition] to infer it from.
  [resolver version](resolver.md#resolver-versions).
- Commands run in the a virtual workspace's directory will run against all workspace
  members by default, see [Package selection](#package-selection).
```

Reply:
I'm not quite seeing the problem.

When I look at the previous paragraph, it ends on the note of the role of virtual workspaces contrasted with root packages

> This is typically useful when there isn't a "primary" package, or
you want to keep all the packages organized in separate directories.

I tweaked the leading line for the new section a little bit which now reads

> By having a workspace without a root package,

Could you help me understand what challenge you are seeing in the transition?

Reply:
Thanks for that. It feels better now. I guess it is because that there is an extra term “primary package”, so root package is a bit far when mentioned.